pkg/arvo/app/pool-group-hook.hoon

```diff
@@ -141,7 +140,7 @@
 ++  new-tid
   |=  eny=@uv
   ^-  @t
-  :((cury cat 3) dap.bowl '--' (scot %uv eny))
```

Comment:
It needs to be `(scot %ta :((cury cat 3) dap.bowl '_' (scot %uv eny)))`.  The `0v1.` part is what doesn't parse.

Reply:
Huh, weird oops here. Thanks for catching. (: